### PR TITLE
update changelong

### PR DESCRIPTION
**Pull request by rsokl**
@jgbos please proofread this and let me know what you think

### CODE REVIEW

File: docs/source/changes.rst [hunk 1]
```diff
@@ -12,3 +12,9 @@ All previous releases should still be available on pip.
 ------------------
 
 This is hydra-zen's first stable release on PyPI!
+Although we have not yet released version `v1.0.0`, it should be noted that hydra-zen's codebase is thoroughly tested.
+Its test suite makes keen use of the property-based testing library `Hypothesis <https://hypothesis.readthedocs.io/en/latest/>`_. 
+Furthermore, 100% code coverage is enforced on all commits into `main`.
+
+We plan to have aggressive release schedule for compatibility-preserving patches of bug-fixes and quality-of-life improvements (e.g. improved type annotations).
```
Comment (rsokl) [inline docs/source/changes.rst:19]:
```suggestion
We plan to have an aggressive release schedule for compatibility-preserving patches of bug-fixes and quality-of-life improvements (e.g. improved type annotations).
```

File: docs/source/changes.rst [hunk 2]
```diff
@@ -12,3 +12,9 @@ All previous releases should still be available on pip.
 ------------------
 
 This is hydra-zen's first stable release on PyPI!
+Although we have not yet released version `v1.0.0`, it should be noted that hydra-zen's codebase is thoroughly tested.
+Its test suite makes keen use of the property-based testing library `Hypothesis <https://hypothesis.readthedocs.io/en/latest/>`_. 
+Furthermore, 100% code coverage is enforced on all commits into `main`.
+
+We plan to have an aggressive release schedule for compatibility-preserving patches of bug-fixes and quality-of-life improvements (e.g. improved type annotations).
+hydra-zen will maintain wide a window of compatibility with Hydra versions; we test against pre-releases of Hydra and will maintain compatibility with its future releases.
```
Comment (jgbos) [inline docs/source/changes.rst:20]: hydra-zen will maintain a window of compatibility with Hydra versions; we test against pre-releases of Hydra and will maintain compatibility with its future releases.

Comment (jgbos) [reply docs/source/changes.rst:20]: wait, i'm new to adding the commit in review.  I was just trying to update the last sentence.

Comment (rsokl) [reply docs/source/changes.rst:20]: Use this button to suggest an edit: 
image

**Review by jgbos — APPROVED** [review]

**Pull request merged**